lib/diplomat/key_utils.ex

```diff
@@ -0,0 +1,200 @@
+defmodule KeyUtils do
```

Comment:
Here is how `urlsafe` is implemented in Python NDB

https://github.com/GoogleCloudPlatform/datastore-ndb-python/blob/master/ndb/key.py#L553